### PR TITLE
Find/replace of `master` to `main` for branch rename

### PR DESCRIPTION
**Pull request by andschwa**
Since `master` is now `main`.

### CODE REVIEW

File: module/PowerShellEditorServices.VSCode/PowerShellEditorServices.VSCode.psd1 [hunk 1]
```diff
@@ -102,7 +102,7 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/master/LICENSE'
+            LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```
Comment (SeeminglyScience) [inline module/PowerShellEditorServices.VSCode/PowerShellEditorServices.VSCode.psd1:105]:
```suggestion
        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```

Think the indent here was unintentional

File: module/PowerShellEditorServices/Commands/PowerShellEditorServices.Commands.psd1 [hunk 2]
```diff
@@ -108,7 +108,7 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/master/LICENSE'
+            LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```
Comment (SeeminglyScience) [inline module/PowerShellEditorServices/Commands/PowerShellEditorServices.Commands.psd1:111]:
```suggestion
        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```

File: module/PowerShellEditorServices/PowerShellEditorServices.psd1 [hunk 3]
```diff
@@ -107,7 +107,7 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/master/LICENSE'
+            LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```
Comment (SeeminglyScience) [inline module/PowerShellEditorServices/PowerShellEditorServices.psd1:110]:
```suggestion
        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```

**Review by SeeminglyScience — CHANGES_REQUESTED** [review]
Just fixing some indent changes that don't seem intentional

File: module/PowerShellEditorServices.VSCode/PowerShellEditorServices.VSCode.psd1 (hunk 1 above)
Comment (andschwa) [reply module/PowerShellEditorServices.VSCode/PowerShellEditorServices.VSCode.psd1:105]: Thanks, super weird, don't know why Code did that. All I did was click the replace button 🤦

**Review by SeeminglyScience — APPROVED** [review]
LGTM!

**Pull request merged**